input/blog/2018-07-06-cake-v0.29.0-released.md

```diff
@@ -0,0 +1,119 @@
+
+```csharp
+var target = Argument("target", "Default");
+
+Task("Dependency-Task")
+    .Does(() =>
+{
+    Information("Executing dependency.");
+});
+
+
+Task("Dependent-Task")
+    .IsDependentOn("Dependency-Task")
+    .Does(() =>
+{
+   Information("Executing task.");
+});
+
+RunTarget(target);
+```
+
+Normally when executing `Cake.exe --target="Dependent-Task"` we would see something like:
+```
+========================================
+Dependency-Task
+========================================
+Executing dependency.
+
+========================================
+Dependent-Task
+========================================
+Executing task.
+
+Task                          Duration
+--------------------------------------------------
+Dependency-Task               00:00:00.0061407
+Dependent-Task                00:00:00.0002070
+--------------------------------------------------
+Total:                        00:00:00.0063477
+```
+
+But when specifying the `exclusive` option `Cake.exe --target="Dependent-Task" --exclusive`, then only the specified task will execute.
+
+```
+========================================
+Dependent-Task
+========================================
+Executing task.
+
+Task                          Duration
+--------------------------------------------------
+Dependent-Task                00:00:00.0059713
+--------------------------------------------------
+Total:                        00:00:00.0059713
+```
+
+### Issues
+
+As part of this release we had [12 issues](https://github.com/cake-build/cake/milestone/49?closed=1) closed.
+
+__Feature__
+
+- [__#2203__](https://github.com/cake-build/cake/issues/2203) Add Octopus Deploy Promote release support.
+- [__#2095__](https://github.com/cake-build/cake/issues/2095) Add "--skipnontestassemblies" functionality to CAKE's NUnit3Settings as it exists in original nunit3 test runner.
+- [__#2094__](https://github.com/cake-build/cake/issues/2094) Add support for executing a single task without dependencies.
+
+__Improvement__
+
+- [__#2196__](https://github.com/cake-build/cake/issues/2196) NuGet Repository information not settable in NuGet Pack.
+- [__#2185__](https://github.com/cake-build/cake/issues/2185) Try to find vswhere.exe on the system if the tool is not registered.
+- [__#2154__](https://github.com/cake-build/cake/issues/2154) Problem with loading absolute path scripts with #load preprocessor.
+- [__#2152__](https://github.com/cake-build/cake/issues/2152) try resolve vstest.console.exe before guessing it.
+- [__#1609__](https://github.com/cake-build/cake/issues/1609) Add additional VSTS actions.
+
+__Documentation__
+
+- [__#2195__](https://github.com/cake-build/cake/pull/2195) Updated the WiX tool documentation.
+- [__#2193__](https://github.com/cake-build/cake/issues/2193) Add Pascal and Dave to all required places.
+- [__#2188__](https://github.com/cake-build/cake/issues/2188) The CLA link in readme seems invalid or broken.
+
+__Breaking change__
```

Comment:
Shouldn't breaking changes be at the top of the release notes?

Reply:
Sure I can fix order.

Reply:
Order fixed.